### PR TITLE
pkg/libinit: migrate from syscall to golang.org/x/sys

### PR DESCRIPTION
**Pull request by orangecms**
Signed-off-by: Daniel Maslowski <info@orangecms.org>

### CODE REVIEW

**Review by rminnich — COMMENTED** [review]
could you add to the comment about what this makes better, since I suspect it's partly "we are adding bsd support"

**Comment by orangecms** [comment]
> could you add to the comment about what this makes better, since I suspect it's partly "we are adding bsd support"

Just a coincidental cleanup. `syscall` has been deprecated in 1.4 already.
And we're already using golang.org/x/sys anyway, so we should get to the point where we're using it everywhere.

See https://pkg.go.dev/syscall

File: pkg/libinit/proc_linux.go [hunk 1]
```diff
@@ -5,21 +5,21 @@
 package libinit
 
 import (
+	"golang.org/x/sys/unix"
 	"log"
 	"os"
 	"os/exec"
-	"syscall"
```
Comment (rminnich) [inline pkg/libinit/proc_linux.go:11]: run gofmt, this "unix" line is in the wrong place.

Comment (hugelgupf) [reply pkg/libinit/proc_linux.go:11]: gofmt doesn't fix this, you need to add an empty line after the stdlib imports and then the unix import. Gofmt only sorts imports within each block of imports separated by empty lines

**Comment by orangecms** [comment]
superseded by #2399 - `plan9` cannot be fully used for this, so it applies merely to `unix`

**Pull request closed without merging**